# DP: hotshot: Check for the availability of the profile and pstats modules.

Index: b/Lib/hotshot/stats.py
===================================================================
--- a/Lib/hotshot/stats.py	2011-07-26 15:35:06.147361347 +0400
+++ b/Lib/hotshot/stats.py	2011-07-26 15:36:54.848054975 +0400
@@ -1,7 +1,10 @@
 """Statistics analyzer for HotShot."""
 
-import profile
-import pstats
+try:
+    import profile
+    import pstats
+except ImportError, e:
+    raise ImportError, str(e) + '; please install the python-profiler package'
 
 import hotshot.log
 
